src/app/components/badges/full-list-of-achievements/full-list-of-achievements.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
+import { Achievelist } from 'src/app/models/achievelist';
+import { AhievementListService } from 'src/app/shared/services/ahievement-list.service';
+
+@Component({
+  selector: 'app-full-list-of-achievements',
+  templateUrl: './full-list-of-achievements.component.html',
+  styleUrls: ['./full-list-of-achievements.component.scss']
+})
+export class FullListOfAchievementsComponent implements OnInit {
+
+  list:Achievelist[] | undefined;
+
+  constructor(private readonly service:AhievementListService) { }
```

Comment:
service is bad variable name, if you would have 2 or more services this would make no sense.,better use variable that represents the instance correctly, in this case it would be 'achievementService'